shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
This highlights one convenient property of `window.confirm()`: you don't have to figure where to root it in the DOM.